snippets/product-variant-picker.liquid

```diff
@@ -0,0 +1,97 @@
+{% comment %}
+    Renders product variant-picker
+
+    Accepts:
+    - product: Product object.
+    - block_context: {Object} either passing the block/section.
+    - section: {String} section name.
+
+    Usage:
+    {% render 'product-variant-picker', block_context: block, product: product %}
+{% endcomment %}
+
+{%- unless product.has_only_default_variant -%}
+  {%- if block_context.settings.picker_type == 'button' -%}
+    <variant-radios
+      id="variant-radios-{{ section.id }}"
+      class="no-js-hidden"
+      data-section="{{ section.id }}"
+      data-url="{{ product.url }}"
+      {{ block_context.shopify_attributes }}
+    >
+      {%- for option in product.options_with_values -%}
+        <fieldset class="js product-form__input">
+          <legend class="form__label">{{ option.name }}</legend>
+          {% render 'product-variant-options', product: product, option: option, block: block_context %}
+        </fieldset>
+      {%- endfor -%}
+      <script type="application/json">
+        {{ product.variants | json }}
+      </script>
+    </variant-radios>
+  {%- else -%}
+    <variant-selects
+      id="variant-selects-{{ section.id }}"
+      class="no-js-hidden"
+      data-section="{{ section.id }}"
+      data-url="{{ product.url }}"
```

Comment:
🤔 I'm wondering why we even need this `data-update-url="false"` in the first place. I don't see us using it anywhere or having something depending on it. @ludoboludo @kmeleta  please correct me if I'm wrong. Thanks.

Reply:
Yeah it's used for the share button when changing the variants. If this is on then the JS that runs while changing the variant won't update the `url` shown in the share button. 
Since it's a data attribute, in order to find out if it's used in the JS you'd need to search for `.updateUrl` 👍 ([Mozilla's doc about naming conversion](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dataset#name_conversion))

Reply:
Thanks Ludo!